spk/firebird/src/wizard/install_uifile

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "step_title": "Configuration of FirebirdSQL server",
+    "items": [
+      {
+        "type": "textfield",
```

Comment:
please use type password and document the default value (or leave it blank)

Reply:
Done. Default value blank.